src/Umbraco.Web/HealthCheck/Checks/Config/TrySkipIisCustomErrorsCheck.cs

```diff
@@ -38,7 +38,7 @@ public override IEnumerable<AcceptableConfiguration> Values
         {
             get
             {
-                var recommendedValue = _serverVersion >= new Version("7.5.0")
+                var recommendedValue = _serverVersion >= new Version("7.5")
```

Comment:
@nul800sebastiaan Yes, its was a little confusing trying to work out why the value was higher when it looked correct! It's not a good day in the office if you don't learn anything new ;-)

For the translation I'm still a little unsure if its best to remove the variable we don't use or keep it for consistency. There is already a precedent set in the checkErrorMessageUnexpectedValue message which only uses 3 of the 4 variables passed in. 

It might be a wider debate around how the keys are documented. Should the comments be the final point of reference or should we expect someone translating to look in the core to see which values they have access to? My personal view is its better to have the extra variable passed in and maintain the consistency around the comments.